File: validation/Prettyprint/omp.f
      program p
      real a(10,10,10,10,10)

      i = 0

      if(i.gt.0) then
         print *, 'RETURN 1 in FOO'
         i = i + 1
      endif

      i = i + 1

      if(i.gt.0) then
         i = i + 1
      endif

      do m=2,5
         do l=5,7
            do k=1, 2                      
               do j=1, 2
                  do i=1, 10
                     x = 2.3
C dsgsgsgs
C frfrfr
                     y=                 5.3 + x
C frfrfr
C frfrfr
C frfrrf
                     z=2.32 + y

                     w=1.2 + z
                     a(i,j,k,l,m) = x*x+x+z
                  enddo
               enddo
            enddo
         enddo
      enddo

      if(w.gt.0) then
         print *, 'RETURN 1 in FOO'
      endif

C troto   
      i= i + 1

      end

      subroutine foo()
C      print *, "foo is entered with ", i
C      if(i.gt.0) then
C         print *, 'RETURN 1 in FOO'
C      endif
      i = i + 1
      end

File: validation/Prettyprint/omp.c

int foo (void) {
  return 0;
}

int main () {
  float a[10][10][10][10][10];
  int i,j,k,l,m;
  float x = 2.12;
  float w = 1.0;

  for (i = 0; i < 10; i++) {
    for (j = 2; j < 7; j++) {
      for (k = 4; k < 3; k++) {
	for (l = 2; l < 10; l++) {
	  for (m = 0; m < 10; m++) {
	    float y;
	    float z=2.32 + y;
	    y = 3.5 + x;	    
	    w=1.2 + z;
	    a[i][j][k][l][m] = x*y+x+z;
	  }
	}
      }
    }
  }
  return 0;
}
